salt/metalk8s/kubeadm/init/kubelet-start/files/service-kubelet-systemd.conf

```diff
@@ -0,0 +1,10 @@
+# Note: This dropin only works with kubelet v1.11+
```

Comment:
Same as above: https://github.com/scality/metalk8s/blob/development/2.0/docs/requirements.rst#tagged-generated-files
I know the generated file will then not be 100% equal to the one dropped by the `kubeadm` package, but that's fine.